### PR TITLE
update `log_resource_info!` to use `log_values!`

### PR DESCRIPTION
**Pull request by ericphanson**
A small optimization for loggers which are more efficient when you batch logging calls (e.g. wandb). Should have no functional change.

### CODE REVIEW

**Review by hannahilea — APPROVED** [review]

File: src/learn.jl [hunk 1]
```diff
@@ -32,7 +31,7 @@ Logs a series plot to `logger` under `field`, where...
 - `curves` is an iterable of the form `Tuple{Vector{Real},Vector{Real}}`, where each tuple contains `(x-values, y-values)`, as in the `Lighthouse.EvaluationRow` field `per_class_roc_curves`
 - `labels` is the class label for each curve, which defaults to the numeric index of each curve.
 """
-log_line_series!(logger, field::AbstractString, curves, labels=1:length(curves))
+log_line_series!(logger, field::AbstractString, curves; labels=1:length(curves))
```
Comment (hannahilea) [inline src/learn.jl:34]: ...is this intentional? Seems better, but also might break downstream consumers (although we suspect that that is exactly one consumer right now, so maybe not a big deal....)

Comment (josephsdavid) [reply src/learn.jl:34]: hmm in LWB we have a comma there:

```julia
"""
    Lighthouse.log_line_series!(::WandbLogger, field::AbstractString, curves, labels=nothing; kwargs...)
Log custom curves to wandb
Where...
    - `field::AbstractString` is the key to log the curves under
    - `curves` is an iterable of the form `Tuple{Vector{Real},Vector{Real}}`, where each tuple contains `(x-values, y-values)`, as in the `Lighthouse.EvaluationRow` field `per_class_roc_curves`
    - `labels` is the class label for each curve. If `nothing`, generates default labels that are the index of each curve
    - `kwargs` contains extra arguments for `wandb.plot.line_series`
"""
function Lighthouse.log_line_series!(::WandbLogger, field::AbstractString, curves,
                                     labels=nothing; kwargs...)
    return log_to_wandb!(field, make_wandb_line_series(curves, labels; kwargs...))
end
```

**Review by hannahilea — CHANGES_REQUESTED** [review]
(see comment!)

File: src/learn.jl [hunk 2]
```diff
@@ -108,10 +108,11 @@ end
 
 function log_resource_info!(logger, section::AbstractString, info::ResourceInfo;
                             suffix::AbstractString="")
-    log_value!(logger, section * "/time_in_seconds" * suffix, info.time_in_seconds)
-    log_value!(logger, section * "/gc_time_in_seconds" * suffix, info.gc_time_in_seconds)
-    log_value!(logger, section * "/allocations" * suffix, info.allocations)
-    log_value!(logger, section * "/memory_in_mb" * suffix, info.memory_in_mb)
+    log_values!(logger,
```
Comment (josephsdavid) [inline src/learn.jl:111]: I really like this idea!

File: src/learn.jl (hunk 1 above)
Comment (ericphanson) [reply src/learn.jl:34]: Oops! I did not mean to change this in this PR, but I think it was supposed to be a comma... Not sure what happened 🤔

Comment (josephsdavid) [reply src/learn.jl:34]: not sure the pros and cons of having the labels as a positional or kw argument!

Comment (josephsdavid) [reply src/learn.jl:34]: ah i misunderstood!

**Review by hannahilea — APPROVED** [review]

**Pull request merged**